iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/TieLineImpl.java

```diff
@@ -229,9 +234,12 @@ public HalfLineImpl getHalf(Side side) {
         }
     }
 
+    // Half1 and half2 are lines, so the transmission impedance of the equivalent branch is symmetric
     @Override
     public double getR() {
-        return half1.getR() + half2.getR();
+        LinkData.BranchAdmittanceMatrix adm = HalfLineUtil.equivalentBranchAdmittanceMatrix(half1, half2);
```

Comment:
@miovd and @zamarrenolm these calculations are duplicated in each implementations and quite complex to maintain for non expert people. Is there a way to put that in the `TieLine` API?